libs/application/templates/aosh/practical-exam/src/fields/ExamCategoriesSelf/index.tsx

```
@@ -0,0 +1,85 @@
+    defaultValue: [],
+  })
+
+  const watchedCategories: string[][] = useWatch({
+    name: 'examCategories',
+  })
+
+  useEffect(() => {
+    ;(async () => {
+      try {
+        const isNextEnabled = await trigger('examCategories')
+        setSubmitButtonDisabled && setSubmitButtonDisabled(!isNextEnabled)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)